### ISSUE TITLE
Eldrich-specific data normalizer/denormalizer

### ISSUE THREAD

**Issue by eaton** [nerdhaus/eldrich#42]

**Comment by EmperorArthur** [comment]
I'd say something like a JSON format.  Something like
```
{
  "attributes": [
    "COG": number;
    "COO": number;
    "INT": number;
    "REF": number;
    "SAV": number;
    "SOM": number;
    "WIL": number;
    "MOX": number;
  ],
  "skills": [
    ...
  ]
}
```


That's probably what I'm going to implement as the export format from the [Character Creator](https://github.com/emperorarthur/ep-character-creator).

**Comment by eaton** [comment]
Yep — in Drupal, that stuff gets handled by the standard JSON serializer, but it can be swapped out for XML, HAL, etc. The Normalizer is the code responsible for building the structure that will ultimately be serialized *into* JSON/etc.

The biggest question marks structurally revolve around weapons, vehicles, and weird edge cases like specialty multi-ammo clips for kinetic weapons. Those shouldn't have too much impact on the character definition itself, though, which is what would matter for inter-operation with the Character Creator…
